kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NM8021xPhase2Auth.java

```diff
@@ -0,0 +1,10 @@
+package org.eclipse.kura.nm.enums;
+
+public enum NM8021xPhase2Auth {
+    eap,
+    mschapv2,
+    gtc,
+    otp, 
+    md5,
+    tls
```

Comment:
Please use all caps enums to be consistent with the already established convention and to make Sonar happy.

See https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NMSettingIP6ConfigPrivacy.java and https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/enums/NMSettingIP6ConfigAddrGenMode.java for reference